File: LercWrapper/LercWrapper.m
//
//  LercWrapper.m
//  LercSwiftMixed
//
//  Created by charlotte on 2018/1/29.
//  Copyright © 2018年 gago. All rights reserved.
//

#import <Foundation/Foundation.h>
#import <Cocoa/Cocoa.h>
#import "LercWrapper.h"

@implementation LercWrapper : NSObject {

}
- (NSImage*)lerc_decode_wrapper:(int)width height:(int)height {
  NSImage* image = [NSImage alloc];
  image.size = NSMakeSize(width, height);
  return image;
}
@end

File: LercWrapper/LercWrapper.cpp
//
//  LercWrapper.cpp
//  LercSwiftMixed
//
//  Created by charlotte on 2018/1/29.
//  Copyright © 2018年 gago. All rights reserved.
//

#include "LercWrapper.hpp"
#include <iostream>
#include <stdio.h>
#include <cstring>
#include <cstdlib>
#include <cstdio>
#include <cmath>
#include <vector>
#include <unistd.h>
#include "Lerc_c_api.h"
using namespace std;

unsigned char* lerc_getBlobInfo_wrapper()
{
  return new unsigned char[4 * 2048 * 2048];
}
